### DISCUSSION TITLE
Change name of rows and columns in highspy

### DISCUSSION THREAD

**Discussion by FabianHofmann** [ERGO-Code/HiGHS#1032] [Ideas]
I am wondering whether there is the possibility to change the names of the rows and columns through the highspy interface. 

Minimal Example
```python 
import highspy
import scipy.sparse

h = highspy.Highs()
h.addVars(2, [0, -10], [20, 0])
h.changeColsCost(2, [0, 1], [5, 4])
A = scipy.sparse.csr_array([[1, -3], [-2, 1]])
num_cons = A.shape[0]
lower = [-10, -20]
upper = [10, 10]
h.addRows(num_cons, lower, upper, A.nnz, A.indptr, A.indices, A.data)
h.writeModel('example.mps')
```

The program then warns:
```
WARNING: There are empty or excessively-long column names: using constructed names with prefix "c"
WARNING: There are empty or excessively-long row names: using constructed names with prefix "r"
```

There is the possibility to change the names in the LP model like 
```python 
lp = h.getLp()
lp.row_names_ = ["c1", "c2"]
lp.col_names_ = ["x1", "x2"]
```
but that does not change anything when writing to file afterwards. Could you help me out here?

**Comment by jajhall** [comment]
Changing the names in a model is not currently possible, even with the C++ class. The only way to define names in a model that's not read from a file, is to create an instance of the `HighsLp` class, and populate its names. 

In your case, as you're defining the LP by adding variables and constraints, once you've done this the route would be 

lp = h.getLp()

populate its names, and then call 

h.passModel(lp)

before 

h.writeModel('example.mps')

I'll turn this into a discussion, as it's not a bug

**Comment by FabianHofmann** [comment]
Thanks @jajhall that makes sense.
